website/docs/en/plugins/dev/hooks.mdx

```diff
@@ -252,6 +252,9 @@ const myPlugin = () => ({
         },
       };
 
+      // extraConfig will override fields in userConfig,
+      // If you do not want to override the fields in userConfig,
```

Comment:
The type should be updated too:

![image](https://github.com/web-infra-dev/rsbuild/assets/7237365/9e4e7d52-87c1-4cf5-a541-2d2d75c2ab8c)